README.md

```diff
@@ -47,3 +47,44 @@ There are three steps to the CI workflow:
    [kustomize](https://kustomize.io/) to updating the image digest in the
    service's respective `/manifests/<SERVICE_NAME>/kustomization.yaml` file
    (only run on master currently - unless we want to add overlays/multiple envs)
+
+### WIP: deployment to kubernetes
+
+Create a kubernetes cluster somewhere
+
+```bash
+gcloud container clusters create dapr --num-nodes=1
+```
+
+Install Dapr on the cluster
+
+```bash
+dapr init -k
+```
+
+Install the Hello Kubernetes Quickstart example Node App. <node.yaml> is https://github.com/dapr/quickstarts/blob/master/hello-kubernetes/deploy/node.yaml. This also installs the Dapr side car, as defined in the yaml
+
+```bash
+kubectl apply -f <node.yaml>
+```
+
+Install the nginx ingress controller
+```bash
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm repo update
+helm install nginx-ingress ingress-nginx/ingress-nginx -f ./manifests/ingress-controller.yaml -n default
+```
+
+Install the ingress rules. These forward to the dapr sidecar of the ingress controller above, which is called "nginx-ingress-dapr" ("-dapr" is added to the names of things to form the sidecar name)
+
+```bash
+kubectl apply -f ./manifests/ingress.yaml
+```
+
+Call the node app via dapr. The external Ip address of the ingress controller can be found from  `kubectl get services`.
+
+```
+curl http://<external-ip-address-of-ingress-controller>/v1.0/invoke/nodeapp/method/ports
```

Comment:
with my below comment, could this become `curl http://<external-ip-address-of-ingress-controller>/ping`?

Reply:
Hi David, I think this would be better in a follow up PR. The nodeapp referenced here isn't something we have written (it comes from the Dapr examples). This PR is already quite big, and the branch has already spent quite a long time not being continously integrated. I think there might be some niggles getting the slack service ping endpoint to work, and these niggles would probably be best all kept together in the same PR, without getting confused by the general changes here that are just getting anything to work.

Let me know if your thinking is different ...

Reply:
True. I suppose as long as there is a ticket in the backlog for making sure everything is appropriately routed and locked down, that's probably fine.

Reply:
I've added a task / ticket to the kanban board for this